src/Illuminate/Http/Client/Factory.php

```diff
@@ -42,15 +43,20 @@ class Factory
      * @var array
      */
     protected $responseSequences = [];
+    /**
+     * @var Dispatcher|null
```

Comment:
Please use fully qualified class names in phpdoc.

Reply:
It's okay, and I use it in the PendingRequestEvent File too